reV/supply_curve/exclusions.py

```diff
@@ -77,7 +86,7 @@ def __init__(self, layer, inclusion_range=(None, None),
         self._mask_type = self._check_mask_type()
 
     def __repr__(self):
-        msg = ("{} for {} exclusion, of type {}"
+        msg = ('{} for "{}" exclusion, of type "{}"'
```

Comment:
one thing I like to do instead of `"{}"` is `{!r}`, since it automatically adds quotation marks to string inputs. This is fine though!

Reply:
Yeah but not for numbers though so i kind of like the explicit quotation marks. I mean I think either is fine really, im going to leave it out of stubbornness :)

Reply:
No objections my friend!